data/json/itemgroups/guns.json

```diff
@@ -133,10 +133,11 @@
     "id" : "guns_rifle_common",
     "//" : "Rifles commonly owned by citizens and found in many locations.",
     "items": [
-      [ "browning_blr", 25 ],
-      [ "garand", 65 ],
       [ "ar15", 30 ],
+      [ "browning_blr", 25 ],
+      [ "cane_rifle", 10 ],
```

Comment:
I thought gun laws were relaxed in the setting? If they are identical to present laws, I will remove the rifle_common, but then several other odd/illegal weapons will need to be moved, like the fully automatic TEC-9, which wasn't ever mass produced as an automatic weapon.